src/quacc/runners/ase.py

```diff
@@ -362,3 +369,171 @@ def _set_sella_kwargs(self, optimizer_kwargs: dict[str, Any]) -> None:
 
         if not self.atoms.pbc.any() and "internal" not in optimizer_kwargs:
             optimizer_kwargs["internal"] = True
+
+
+def run_neb(
+    images,
+    relax_cell: bool = False,
+    fmax: float = 0.01,
+    max_steps: int | None = 1000,
+    optimizer: NEBOptimizer | BFGS = NEBOptimizer,
+    optimizer_kwargs: OptimizerKwargs | None = None,
+    run_kwargs: dict[str, Any] | None = None,
+    neb_kwargs: dict[str, Any] | None = None,
+    copy_files: SourceDirectory | dict[SourceDirectory, Filenames] | None = None,
+) -> list[Atoms]:
```

Comment:
At some point before merging, this function needs to be a method within the `Runner` class as you see for the other methods in this module. I can assist with that later, although you can of course try yourself if you wish.

Reply:
Did that.